.github/workflows/publish_docker_base.yml

```diff
@@ -8,6 +8,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Do we need/want to keep this now that you've triggered the build manually?

Reply:
Yeah I think it's still useful to keep around, for the next time we want to trigger it manually.